contrib/completion/bash/docker

```diff
@@ -70,9 +85,67 @@ __docker_container_ids() {
 	COMPREPLY=( $(compgen -W "${containers[*]}" -- "$cur") )
 }
 
+__docker_image_generate_completions() {
+	local counter=$( __docker_pos_first_nonflag $( __docker_to_alternatives "$options_with_args" ) )
+
+	if [ $cword -eq $counter ]; then
+
```

Comment:
I think position checking should not be moved into this function.
It's a different concern and also introduces a coupling to the number of images that can be completed (just one).